cmd/http/main.go

```diff
@@ -61,7 +65,19 @@ func main() {
 	}
 	defer nc.Close()
 
-	conn, err := grpc.Dial(cfg.ThingsURL, grpc.WithInsecure())
+	secureOption := grpc.WithInsecure()
```

Comment:
@nwest1 I would put this in a separate function as you did here: https://github.com/mainflux/mainflux/blob/897d4373b15b5fbd7b346086a3552929c3fedef4/cmd/things/main.go#L156

Reply:
Done!